link-grammar/api.c

```diff
@@ -1403,10 +1409,6 @@ int sentence_parse(Sentence sent, Parse_Options opts)
 {
 	int rc;
 
-	verbosity = opts->verbosity;
-	debug = opts->debug;
-	test = opts->test;
```

Comment:
This was an old workaround for this initialization bug (that this pull request fixes).

The globals are actually changed in the corresponding parse_options_set*() functions:
```
void parse_options_set_verbosity(Parse_Options opts, int dummy)
{
	opts->verbosity = dummy;
	verbosity = opts->verbosity;                <--------------------- here
	/* this is one of the only global variables. */
}
```

As you can see, these assignments would be redundant unless the said initialization bug.
They were added only to catch the setup done in parse_options_create(). Afterwards, they always assign to the globals the same value they already have.

Currently (without the pull request fix), these globals don't get their correct values in the dictionary reading code, and also not in a sentence_split() before calling sentence_parse().

So the fix is needed.